recipes/cruft/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "cruft" %}
+{% set version = "1.4.0" %}
+
```

Comment:
Couple of questions/reviews. 

1. cruft 1.4.0 has different dependencies than the ones you have listed below. Master has 2.0 deps but that is pending a pypi release.

2. How is version bumping handled here? Do we need to explicitly update this every time?

Reply:
conda-forge has a bot that checks for updates on pypi, so when a new release happens, a PR should be automatically opened. It won't fix the dependency changes (that has to be done manually), but you can edit the PR or just open a new one. If there aren't any changes needed except for the version bump and sha256 update (which the bot handles automatically), then you can merge once the CI passes.

Reply:
Looking into this today and will fix the dependencies to reflect the PyPI release. Thanks for the clarification, @synapticarbors!